kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -638,6 +639,34 @@
     }
   }
 
+  function initTranslationDraft() {
+    var $draftButton = $('.btn-draft'),
+      url = $('.btn-draft').data('draft-url'),
+      draftMessage = $('#draft-message-success').html();
+    $draftButton.click( function() {
+      var bothData = $('#both_form').serializeArray(),
+        docData = $('#doc_form').serializeArray(),
+        revData = $('#rev_form').serializeArray(),
+        totalData = $.extend(bothData, docData, revData);
+      $.ajax({
+        type: 'POST',
+        url: url,
+        data: totalData,
+        timeout: 3000
+      })
+        .done(function() {
+          var time = new Date(),
+            message = draftMessage + time;
```

Comment:
This way of building a message is bad for localization. You should move the text out of the HTML and into the JS and make this line something like `message = k.safeInterpolate(_('Draft has been saved on: {time}'), {time: time});`.